src/__snapshots__/generator.test.ts.snap

```diff
@@ -0,0 +1,5169 @@
+
+                        set inner(inner: OuterMiddleBBInner) {
+                            this._inner = inner
+                        }
+
+                        encode(buf: Uint8Array): Uint8Array {
+                            let encoded = buf
+                            encoded = this._inner.encode(encoded)
+                            return encoded
+                        }
+
+                        static decode(buf: Uint8Array): {
+                                  buf: Uint8Array,
+                                  value: OuterMiddleBB
+                                } {
+                            let decoded = buf
+                            const inner = OuterMiddleBBInner.decode(decoded)
+                            decoded = inner.buf
+                            return {
+                                  buf: decoded,
+                                  value: new OuterMiddleBB(inner.value)
+                                }
+                        }
+                    }
+
+                    export class OuterMiddleBBInner {
+                        constructor(ival: number, booly: boolean) {
+                            this._ival = ival
+                            this._booly = booly
+                        }
+
+                        private _ival: number;
+
+                        get ival(): number {
+                            return this._ival
+                        }
+
+                        set ival(ival: number) {
+                            this._ival = ival
+                        }
+
+                        private _booly: boolean;
+
+                        get booly(): boolean {
+                            return this._booly
+                        }
+
+                        set booly(booly: boolean) {
+                            this._booly = booly
+                        }
+
+                        encode(buf: Uint8Array): Uint8Array {
+                            let encoded = buf
+                            encoded = encodeInt32(encoded, this._ival)
+                            encoded = encodeBoolean(encoded, this._booly)
+                            return encoded
+                        }
+
+                        static decode(buf: Uint8Array): {
+                                  buf: Uint8Array,
+                                  value: OuterMiddleBBInner
+                                } {
+                            let decoded = buf
+                            const ival = decodeInt32(decoded)
+                            decoded = ival.buf
+                            const booly = decodeBoolean(decoded)
+                            decoded = booly.buf
+                            return {
+                                  buf: decoded,
+                                  value: new OuterMiddleBBInner(ival.value,booly.value)
+                                }
+                        }
+                    }
+
+                    export class SampleMessage {
```

Comment:
I'd prefer if they were treated as a regular message rather than their fields being omitted entirely in output, this is to match the behaviour of the polyglot-go generator

Reply:
That makes sense! Resolved in 3d3772201e2e1c04f006134e6f782e63b3c45e17 and snapshot updated :)